src/test/java/com/ge/predix/uaa/token/lib/HttpServletRequestUtilTest.java

```diff
@@ -107,6 +107,15 @@ public void testGetSubdomainFromOneHeader(final String requestHeader, final List
         Assert.assertEquals(subdomain, subdomainValue);
     }
 
+    @Test(dataProvider = "domainDataProvider")
+    public void testGetSubdomainFromDomain(final String serverName, final List<String> serviceConfigBaseDomains,
```

Comment:
name suggestion for first parameter:   requestHostname
2nd param:   serviceBaseDomains